npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser.rb

```diff
@@ -327,7 +328,7 @@ def package_files
               dependency_files.
               select { |f| f.name.end_with?("package.json") }.
               reject { |f| f.name == "package.json" }.
-              reject { |f| f.name.include?("node_modules/") }.
+              reject { |f| f.name.include?("node_modules/") if Experiments.enabled?(:yarn_berry) }.
```

Comment:
I think we would want to continue ignoring `node_modules/` for projects without yarn_berry enabled too?

Reply:
That's a new change made specifically for the yarn berry changes, so we want to exclude it when the flag is disabled to preserve the old behavior